frontend/src/App.tsx

```diff
@@ -1,34 +1,31 @@
 import './App.css'
 import { Component } from 'react'
 import React from 'react'
-import PieRechartComponent from './components/pie.rechart'
-import PieVictoryComponent from './components/pie.victory'
+import { PieData, PieRechartComponent } from './components/pie.rechart'
+import { PieVictoryComponent } from './components/pie.victory'
+
 
-type PieData = {
-    name: string
-    value: number
-  }[]
```

Comment:
Not needed as you are going to import it from the class file to reduce repeated declarations and possible issues